Gemfile

```diff
@@ -2,11 +2,10 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 source 'https://gems.weblinc.com' do
-  gem 'workarea'
   gem 'workarea-ci'
-  gem 'workarea-gift_cards'
 end
 
+gem 'workarea-gift_cards', github: 'workarea-commerce/workarea-gift-cards', branch: 'GIFTCARDS-7-backport-missing-via-param' # '~> 3.4.7'
```

Comment:
addresses the storefront test failure caused by gift cards being included. can be removed after we release on tuesday.